lib/properties.js

```diff
@@ -61,11 +66,16 @@ Properties.prototype.get = function(target, name) {
     return target.$attrs[name];
   }
 
-  var propertyName = property.name;
+  var propertyName = getPropertyName(target, name, property);
 
   // check if access to collection property and lazily initialize it
   if (!target[propertyName] && property.isMany) {
-    defineProperty(target, property, []);
+    Object.defineProperty(target, propertyName, {
```

Comment:
What is the reason to not use the existing utility?

Reply:
It needs an additional parameters and I felt the function becomes more of an overhead than providing its usefulness as a wrapper with the benefit of standardizing the parameters to be passed. I removed the function in an early development stage (not without some doubt) and just kept it that way.

With your remark and a review, I [added](https://github.com/bpmn-io/moddle/pull/39/commits/e75cba86d545bc7a9a86d39c731671e12f1ba4b3) function `defineProperty()` back with an additional parameter for the property name. There are some implementation options, but I cannot find a way to get rid of this parameter.

I'm still undecided. Please take a look and decide yourself if we should keep it.